sqlalchemy-stubs/sql/selectable.pyi

```diff
@@ -40,13 +40,13 @@ class FromClause(Selectable):
     def correspond_on_equivalents(self, column, equivalents): ...
     def corresponding_column(self, column, require_embedded: bool = ...): ...
     @property
-    def description(self): ...
+    def description(self) -> str: ...
     @property
-    def columns(self): ...
+    def columns(self) -> ImmutableColumnCollection: ...
     @property
-    def primary_key(self): ...
+    def primary_key(self) -> ColumnSet: ...
     @property
-    def foreign_keys(self): ...
+    def foreign_keys(self) -> Set[FromClause]: ...
```

Comment:
These two are quite counterintuitive (and also below), maybe add a link to the docs original code?

Reply:
You're right. This (and below) should be `Set[sqlalchemy.sql.schema.ForeignKey]`.